app/api/utils/fixturesFactory.ts

```diff
@@ -82,6 +85,19 @@ function getFixturesFactory() {
       commonProperties,
     }),
 
+    entityPermission: (
+      user?: string,
+      type?: PermissionSchema['type'],
+      level?: PermissionSchema['level']
+    ) => {
+      const isPublic = !user || user === 'public';
+      return {
```

Comment:
Wasn't this what we were trying to remove? Surely you have good reasons for this, let's talk about it.